.github/workflows/snyk.yml

```diff
@@ -0,0 +1,27 @@
+name: Snyk
+on:
+  push:
+    branches: [main]
+  pull_request:
```

Comment:
Just for my own edification, why `pull_request` rather than the `pull_request_target` event I see in the examples?

Reply:
It's in the commit message.

Reply:
we'll want to change it to something more like:

```yaml
on:
  pull_request_target:
    types: [labeled]

jobs:
 snyk_vanagon:
   runs-on: ubuntu-latest
   if: contains(github.event.pull_request.labels.*.name, 'safe to test')
```

to ensure we're not leaking the snyk token

Reply:
Ah, I see this repo has workflows enabled for external forks. I'd really rather the Snyk integration just worked rather than having to add this and use a label.

Reply:
We can leave the trigger as `pull_request` instead of `pull_request_target` but it won't work for PRs from forks because it won't have access to the github secrets

Reply:
That works for me for the moment.

Reply:
sounds good, I added the key to this repo (not the org)